systers_portal/meetup/models.py

```diff
@@ -45,3 +45,13 @@ def clean_fields(self, *args, **kwargs):
         """Validate meetup date is not less than today's date"""
         if self.date < timezone.now().date():
             raise ValidationError("Date should not be less than today's date.")
+
+
+class Rsvp(models.Model):
+    """ Users RSVP for particular meetup """
+    user_id = models.ForeignKey(SystersUser)
+    meetup_id = models.ForeignKey(Meetup)
+    is_coming = models.BooleanField()
```

Comment:
I assumed that `is_coming` means if the user himself/herself is coming to the meetup.
`plus_one` means that the user is coming and also bringing one extra person with him/her.